ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -64,14 +64,14 @@ spec:
           ptp4lOpts: "-2 -s --summary_interval -4"
           phc2sysOpts: "-a -r -n 24"
     - fileName: SriovOperatorConfig.yaml
-      policyName: "sriov-conf-policy"
+      policyName: "config-policy"
```

Comment:
Why is this group ? Mandatory for all DU deployments.

Reply:
I believe it is in the group because it contains the mapping to a machine config pool and the disableDrain flag. The thinking was that theyse might be different between SNO and standard clusters.

Reply:
Right, although we do not expose the selector in our current reference example,